src/Slider/ScoreSliders.qml

```diff
@@ -83,7 +81,6 @@ ColumnLayout {
                                            "myValue": tmpValue,
                                            "myTo": JSON.stringify(tmpTo),
                                            "myStepSize": tmpStepSize,
```

Comment:
why the "my" prefix for these things ?

Reply:
I have not right answer, i will change this